job.c

```diff
@@ -34,7 +34,7 @@ store_job(job j)
     all_jobs_used++;
 
     /* accept a load factor of 4 */
-    if (all_jobs_used > (all_jobs_cap << 2)) rehash();
+    if (all_jobs_used > (all_jobs_cap >> 1)) rehash();
```

Comment:
Let's put this back how it was. The intention is to accept a [load factor](https://en.wikipedia.org/wiki/Load_factor_(computer_science)) of 4, so we grow the hash table when the number of entries exceeds the 4*capacity.